west.yml

```diff
@@ -139,7 +139,7 @@ manifest:
       groups:
         - hal
     - name: hal_ti
-      revision: 5e7d5cd584047699c9fd279923120cb25ba3dda7
+      revision: pull/45/head
```

Comment:
the `hal_ti` PR has been merged. You can go ahead and updated this to 4e0e053f5ea413ef0cb20000729038d921ac2746

Reply:
I'll pull it in with a rebase.